packages/e2e-tests/fixtures/blocks/core__cover__deprecated-2.serialized.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:cover {"url":"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==","id":34} -->
+<div class="wp-block-cover has-background-dim" style="background-image:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==)"><div class="wp-block-cover__inner-container"></div></div>
```

Comment:
`<strong>Cover Block</strong>` is missing in the output.

Reply:
Now resolved by adding a migration function.